repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphMapper.java

```diff
@@ -196,11 +203,14 @@ public class EntityGraphMapper {
 
     private static final Set<String> excludedTypes = new HashSet<>(Arrays.asList(TYPE_GLOSSARY, TYPE_CATEGORY, TYPE_TERM, TYPE_PRODUCT, TYPE_DOMAIN));
 
+    Configuration configuration = ApplicationProperties.get();
+    KafkaNotification kfknotif = new KafkaNotification(configuration);
+
     @Inject
     public EntityGraphMapper(DeleteHandlerDelegate deleteDelegate, RestoreHandlerV1 restoreHandlerV1, AtlasTypeRegistry typeRegistry, AtlasGraph graph,
                              AtlasRelationshipStore relationshipStore, IAtlasEntityChangeNotifier entityChangeNotifier,
                              AtlasInstanceConverter instanceConverter, IFullTextMapper fullTextMapperV2,
-                             TaskManagement taskManagement, TransactionInterceptHelper transactionInterceptHelper) {
+                             TaskManagement taskManagement, TransactionInterceptHelper transactionInterceptHelper) throws AtlasException {
```

Comment:
What necessitated this method's signature change?

Reply:
`KafkaNotification` throws the `AtlasException` and since it is being used here, the signature was complaining about this missing exception hence the change

Reply:
But `KafkaNotification` isn't being called in `EntityGraphMapper` (the func where the signature has been changed)?

`kfknotif` is being called in `propagateClassification` though. Could you clarify this?